pages/docs/quickstart.mdx

```diff
@@ -1,9 +1,14 @@
 import DocsLayout from "../../components/layouts/DocsLayout.js";
 import ClipboardCopy from "../../components/ClipboardCopy.js";
 import Link from "../../components/Link.js";
+import QuickstartVideo from "../../components/QuickstartVideo.js";
+
 
 ## Quickstart
 
+<QuickstartVideo>
```

Comment:
I think if it is going to be a new component, the video url should be a prop so we can reuse the component in other areas in the docs, as we have more doc videos coming out

Reply:
I thought that for future use of the same component. Thank you for feedback.